test/data/fake_samples.json

```diff
@@ -1,5 +1,5 @@
-	{"id": "d4789f36-6148-45ca-83c7-1418c037ecbe", "user": "slebras", "name": "s2", "node_tree": [{"id": "s2", "type": "BioReplicate", "parent": null, "meta_controlled": {"depth": {"value": 1.04, "units": "cm"}, "country": {"value": "Uzbekistan"}, "elevation_start": {"units": "feet", "value": 3}}, "meta_user": {"elevation_start": {"units": "feet", "value": 3}, "sesar:material": {"value": "Soil"}, "depth": {"value": 1.04}, "country": {"value": "Uzbekistan"}, "user_field": {"value": "blip"}}, "source_meta": [{"key": "depth", "skey": "Depth", "svalue": {"value": 1.04}}, {"key": "country", "skey": "Country", "svalue": {"value": "Uzbekistan"}}, {"key": "elevation_start", "skey": "elevation start", "svalue": {"value": 3}}]}], "save_date": 1619647313130, "version": 1},
-	{"id": "af9e85f1-32b2-4ddc-90c6-0101c18a8751", "user": "slebras", "name": "s3", "node_tree": [{"id": "s3", "type": "BioReplicate", "parent": null, "meta_controlled": {"depth": {"value": 299, "units": "cm"}, "country": {"value": "Cambodia"}, "biome": {"value": "ENVO:01000221"}, "feature": {"value": "ENVO:00002226"}, "elevation_start": {"units": "meters", "value": 5.9}}, "meta_user": {"elevation_start": {"units": "meters", "value": 5.9}, "depth": {"value": 299}, "country": {"value": "Cambodia"}, "biome": {"value": "ENVO:01000221"}, "feature": {"value": "ENVO:00002226"}, "user_field": {"value": "blap blork"}}, "source_meta": [{"key": "depth", "skey": "Depth", "svalue": {"value": 299}}, {"key": "country", "skey": "Country", "svalue": {"value": "Cambodia"}}, {"key": "biome", "skey": "biome", "svalue": {"value": "ENVO:01000221"}}, {"key": "feature", "skey": "feature", "svalue": {"value": "ENVO:00002226"}}, {"key": "elevation_start", "skey": "elevation start", "svalue": {"value": 5.9}}]}], "save_date": 1619647315141, "version": 1}
-]
\ No newline at end of file
+  {"id": "809c231a-0637-4c3b-b1fb-8cb233d26206", "user": "tgu2", "name": "s1", "node_tree": [{"id": "s1", "type": "BioReplicate", "parent": null, "meta_controlled": {"depth": {"value": 0, "units": "cm"}, "country": {"value": "Germany"}, "biome": {"value": "ENVO:01000221"}, "feature": {"value": "ENVO:01000002"}}, "meta_user": {"sesar:elevation_start": {"value": 123}, "country": {"value": "Germany"}, "feature": {"value": "ENVO:01000002"}, "biome": {"value": "ENVO:01000221"}, "depth": {"value": 0}, "sesar:material": {"value": "Gas"}, "elevation_unit": {"value": "cm"}}, "source_meta": [{"key": "depth", "skey": "Depth", "svalue": {"value": 0}}, {"key": "country", "skey": "Country", "svalue": {"value": "Germany"}}, {"key": "biome", "skey": "biome", "svalue": {"value": "ENVO:01000221"}}, {"key": "feature", "skey": "feature", "svalue": {"value": "ENVO:01000002"}}]}], "save_date": 1621458488627, "version": 1},
```

Comment:
just noticed that this doesn't seem to group the `sesar:elevation_start` and `elevation_unit` fields.
they were previously grouped as: `{"units": "cm", "value": 123}`, but are now two separate entries

Reply:
💯